doc/lsst.prodstatus/quickstart.rst

```diff
@@ -0,0 +1,400 @@
+##########
+prodstatus
+##########
+
+``prodstat`` provides scripts which are used  to organize DP0.2 production and collect production statistics.
+Collected statistics in form of plots and tables can be reported to corresponding Jira tickets.
+
+Obtaining the package -- initial setup
+======================================
+
+::
+
+   setup lsst_distrib
+   git clone https://github.com/lsst-dm/prodstatus.git
+   cd prodstatus
+   setup prodstatus -r .
+   scons  
+
+Notes on Compiling the package the first time and Running tests
+---------------------------------------------------------------
+
+::
+
+   scons
+
+This makes a link between bin.src and the bin dir and compiles anything that needs
+compiling (in this case its pure python so no compiling), it also makes sure the python 
+is setup in the right place this only needs to be run once after checking out or updating
+
+Not all tests may pass, but proceed for now.
+
+After this initial 'scons', for subsequent runs of prodstat commands,
+it is sufficient to run::
+
+  setup lsst_distrib
+  cd prodstatus
+  setup prodstatus -r .
+
+in prodstatus directory
+If you haven't changed any binaries or added any new python files, you don't
+have to run scons again.
+
+Set up the package -- subsequent setups
+=======================================
+
+::
+   
+   setup lsst_distrib ;# if you have not done so aready
+   cd prodstatus  ;#if you are not there already
+   setup prodstatus -r .   
```

Comment:
In fact, if you have to `setup lsst_distrib` beforehand, then something is wrong with the dependencies listed in the `ups/*.table` file.  You should never have to do more than `setup -r .`.

Reply:
What should be the dependency listed in ups/prodstatus.table so that effectively lsst_distrib gets setup when we do a 'setup -r . prodstatus'  (and should 'setup -r . .' work?)

Reply:
Work out what packages you actually use and add those explicitly to the table file. It looks like you are only using ctrl_bps, daf_base (for the old format task metadata), and daf_butler

Reply:
setup -r . prodstatus still requires a jira to be available. Until jira becomes a standard part of the lsst_distrib stack we are asking users to make sure jira is installed locally. We are not yet adding 'setup lsst_distrib' to the EUPS/setup for prodstatus at time time until this issue is resolved.  It may happen once jira goes to tokens rather than a passwd login thru the jira python API.

Reply:
Have you asked for jira to be added to rubin-env-extras? You can file an RFC to request this.

Reply:
https://jira.lsstcorp.org/browse/RFC-873